platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -201,6 +226,67 @@ public void run() {
 				}
 			});
 			return true;
+		} else if ((event.getSource() & InputDevice.SOURCE_MOUSE) == InputDevice.SOURCE_MOUSE) {
+			if (event.getActionMasked() == MotionEvent.ACTION_HOVER_ENTER ||
+					event.getActionMasked() == MotionEvent.ACTION_HOVER_MOVE ||
+					event.getActionMasked() == MotionEvent.ACTION_HOVER_EXIT) {
+				final int x = Math.round(event.getX());
```

Comment:
Here and elsewhere, why not pass a `float` instead of rounding it to an `int`. Seems like we'd be introducing precision errors that can be easily avoided by passing a `float`.

Reply:
I just copied how STYLUS hover event, shouldn't it be changed in both?